src/Raven.Client/ServerWide/Commands/GetNodeInfoCommand.cs

```diff
@@ -22,12 +23,34 @@ public class NodeInfo
         public RachisState CurrentState;
         public bool HasFixedPort;
         public int ServerSchemaVersion;
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(NodeTag)] = NodeTag,
+                [nameof(TopologyId)] = TopologyId,
+                [nameof(Certificate)] = Certificate,
+                [nameof(ClusterStatus)] = ClusterStatus,
+                [nameof(NumberOfCores)] = NumberOfCores,
+                [nameof(InstalledMemoryInGb)] = InstalledMemoryInGb,
+                [nameof(UsableMemoryInGb)] = UsableMemoryInGb,
+                [nameof(BuildInfo)] = BuildInfo,
+                [nameof(OsInfo)] = OsInfo,
```

Comment:
Should be: `OsInfo?.ToJson()` ?

Reply:
Implements IDynamicJson

Reply:
https://github.com/ravendb/ravendb/blob/v4.2/src/Sparrow/Json/Parsing/ObjectJsonParser.cs#L220